packages/element/README.md

```diff
@@ -289,21 +289,19 @@ aside from `children` are passed.
 
 _Parameters_
 
--   _props_ `Object`: 
--   _props.children_ `string`: HTML to render.
--   _props.props_ `Object`: Any additonal props to be set on the containing div.
+-   _props_ `RawHTMLProps`: 
```

Comment:
These sorts of changes make me sad 😕 Definitely a change for the worse.

To be clear, not directed at you, but rather our tooling for documentation not (currently) supporting the following of custom types.

It's the same sort of problem I was running into with #19520, where ultimately I decided to put the entire effort on pause.

Reply:
Agreed… I'm hoping #21238 will improve this situation.

In the case of `RawHTMLProps` a description would likely ameliorate the problem.

Reply:
Unfortunately, the problem isn't solved right now. It needs changes in the pipeline. #21238 is pretty big by now. (119 files changed with +3,450 −8,376) So, I tried not to add new features as much as possible. Because It can make review hard. And it makes merge hard.

But I put it in the later works section to track the ideas and do them later when #21238 is done.

Reply:
👍 Awesome to hear you're already thinking about this!

> But I put it in the later works section

Perfect, please don't add to #21238 🙂

Reply:
I've added a better description here which should improve things.

Reply:
>I've added a better description here which should improve things.

Thanks, it helps me feel better about these changes 👍